contracts/test/integration/utils/TimelockSimulation.sol

```diff
@@ -0,0 +1,96 @@
+pragma solidity =0.8.13;
+
+import {TimelockController} from "@openzeppelin/contracts/governance/TimelockController.sol";
+import {DSTest} from "./../../unit/utils/DSTest.sol";
+import {Vm} from "./../../unit/utils/Vm.sol";
+
+import "hardhat/console.sol";
+
+contract TimelockSimulation is DSTest {
+    /// an array of actions makes up a proposal
+    struct action {
```

Comment:
Struct names should be in the CapWords style: https://docs.soliditylang.org/en/v0.8.14/style-guide.html